exercises/knapsack/instructions.md

```diff
@@ -0,0 +1,25 @@
+# Description
+
+Your task is to determine which items to take to maximize their value, taking into account the knapsack's carrying capacity.
```

Comment:
The "Your task" line was at the bottom of the instructions in a bunch of the other reviews. Should we try to be consistent about putting the "ask" at the top/bottom?

Reply:
It should be at the top.